stages/org.osbuild.coreos.live-artifacts.mono

```diff
@@ -923,7 +926,9 @@ def main(workdir, tree, inputs, options, loop_client):
 
     mk_osmet_files(deployed_tree, img_metal, img_metal4k, loop_client, paths, os_release)
 
-    blsentry_kargs = mkrootfs_metal(paths, workdir, img_metal, fstype, fsoptions, loop_client)
+    (blsentry_kargs, osname) = mkrootfs_metal(paths, workdir, img_metal, fstype, fsoptions, loop_client)
```

Comment:
(super nitpick) I don't think the `()` are needed, this could just be: `blsentry_kargs, osname = mkrootfs_metal(paths...` (we are not super consistent about this in our code though)